### ISSUE TITLE
EVRF entries allowed CRS's

### ISSUE THREAD

**Issue by heidivanparys** [INSPIRE-MIF/helpdesk#27]
The current list of allowed CRS's contains EPSG:5730 (EVRF2000 height). The EPSG registry contains more recent realizations of EVRS: EPSG:5621 (EVRF2007 height) and EPSG:9389 (EVR2019 height). Those last two should probably be added to a future CRS registry too.

**Comment by KathiSchleidt** [comment]
What's the status on adding the updated EVRF codes to those valid for INSPIRE? 

Working for RWS in NL, we've just completed a proof-of-concept on providing elevation data in accordance with the [TG for WCS](https://inspire.ec.europa.eu/id/document/tg/download-wcs) as well as the [Coverage Good Practice](https://inspire.ec.europa.eu/good-practice/ogc-compliant-inspire-coverage-data-and-service-implementation), details at https://github.com/codefornl/INSPIRE-Coverages

One open issue pertains to the vertical CRS, INSPIRE still requires EVRF2000 height while SOTA would imply EVRF2019 height (EPSG::9389) - thus the addition of EVRF2019 would be most welcome!

A pity that the CRS updates are not formally encoded in the CRS data, while the information on such updates is often found in the gml:remarks, this is hard to parse - otherwise such updates could automated whenever updates for the common INSPIRE CRS are published.

**Comment by KathiSchleidt** [comment]
Also - I just noticed that the most current elevation CRS recommended by the NL experts is not on this list - any chance of adding [EVRF2019 mean-tide height epsg-crs-9390](http://www.opengis.net/def/crs/EPSG/0/9390)?
Related - any reason NOT to use this for terrestrial elevation data?

**Comment by heidivanparys** [comment]
EPSG:5621 (EVRF2007 height) and EPSG:9389 (EVR2019 height) are now approved for use in INSPIRE, see https://github.com/INSPIRE-MIF/technical-guidelines/issues/27 and https://github.com/INSPIRE-MIF/helpdesk/blob/main/crs.md.
